tests_v2/test_workspace_icon_actions.py

```
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+"""Regression: workspace icons use the CmuxButtonIcon wire shape end-to-end."""
+
+import glob
+import json
+import os
+import subprocess
+from pathlib import Path
+from typing import Any, Dict, List
```

Ruff (0.15.12)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)